package.json

```diff
@@ -4,31 +4,36 @@
   "private": true,
   "scripts": {
     "dev": "next dev",
-    "build": "next build",
+    "prebuild": "npm run fix && npm run coverage",
+    "build": "next build --no-lint",
     "start": "next start",
-    "lint": "gts lint",
+    "lint": "eslint */*.ts",
+    "fix": "eslint */*.ts --fix",
     "clean": "gts clean",
-    "compile": "tsc",
-    "fix": "gts fix",
-    "prepare": "npm run compile",
-    "pretest": "npm run compile",
-    "posttest": "npm run lint"
+    "coverage": "vitest run * --coverage",
+    "test": "vitest"
```

Comment:
add coverage and testing scripts. `coverage` is meant as part of the build process and will not watch for changes unlike the regular `test` script which is for development